src/Conf.cpp

```diff
@@ -381,17 +392,19 @@ void Conf::setCustomCommand(const ConfParser::Node* node)
         CustomCommandConf::init(cc, p->key.c_str(), Command::Sentinel);
         auto s = p->sub;
         for (;s ; s = s->next) {
-            if (setInt(cc.cmd.minArgs, "MinArgs", s, 2)) {
-            } else if (setInt(cc.cmd.maxArgs, "MaxArgs", s, 2, 9999)) {
-            } else if (setCommandMode(cc.cmd.mode, "Mode", s)) {
+            if (setInt(cc.minArgs, "MinArgs", s, 2)) {
+            } else if (setInt(cc.maxArgs, "MaxArgs", s, 2, 9999)) {
+            } else if (setCommandMode(cc.mode, "Mode", s)) {
             } else {
                 Throw(UnknownKey, "%s:%d unknown key %s", s->file, s->line, s->key.c_str());
             }
         }
-        if (cc.cmd.maxArgs < cc.cmd.minArgs) {
+        if (cc.maxArgs < cc.minArgs) {
            Throw(InvalidValue, "%s:%d must be MaxArgs >= MinArgs", p->file, p->line);
         }
-        Command::addCustomCommand(&cc.cmd);
+        Command cmd;
+        CustomCommandConf::convert(cc, cmd);
+        Command::addCustomCommand(cmd);
     }
```

Comment:
no, Bug still,
Command.name = CustomCommandConf.name.c_str()
the CustomCommandConf.name.c_str() maybe changed after  mCustomCommands.push_back

Reply:
Applied.